### DISCUSSION TITLE
How to change the little avatar/icon on my contributions only from client?

### DISCUSSION THREAD

**Discussion by CouldBeThis** [community/community#22703] [New to GitHub]
<p>How do I change this icon? It seems to be only on contributions made from desktop client whereas contributions from the web interface have the normal photo.</p>
<p><img src="https://user-images.githubusercontent.com/53547181/181072649-ef2f2748-57cc-47a1-a0d9-fd0e1dff360a.png" alt="Screen Shot 2020-08-17 at 10.59.02 AM" data-base62-sha1="wlNRrqff5rz4xLZ6ZfEND1nSTbL" width="371" height="110"></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That most likely means Github doesn’t recognize the author email used in the commit. Depending on whether you really want to use that email address you’ll need to either <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/adding-an-email-address-to-your-github-account">add the email address to your Github account</a>, or <a href="https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#setting-your-commit-email-address-in-git">change the email address your desktop client uses</a>.</p>

**Comment by CouldBeThis** [comment]
<p>aha. you are correct. fixed. thank you!</p>
